_posts/2023-09-22-licensing-update.md

```diff
@@ -0,0 +1,288 @@
+
+1. Allowing downstream codes to freely import or link to the MDAnalysis library with minimal worry about the license they choose to use.
+2. Allowing downstream codes to use and subclass any MDAnalysis components under its application programming interface
+   (classes, methods, and data objects), without incurring copyleft and incurring the license choice of the downstream code.
+3. Ensuring that codes that either directly copy or extend the MDAnalysis library’s source code would fall under the copyleft
+   license requirements of the MDAnalysis library license. Doing so would ensure that eventual license would more closely match the
+   original intent of the historical [GPL v2+](GPL v2) contributions which were kindly donated to MDAnalysis over the years.
+
+It is our view that a weak copyleft license, such as the [GNU Lesser General Public License][LGPL v3] would best match these requirements.
+
+Upon announcing the proposed license change, we engaged in a [consultation period](https://www.mdanalysis.org/2022/11/07/relicensing/#consultation-period-7th-november-until-5th-december-2022)
+where we encouraged members of the community to comment on and ask questions about the proposed relicensing efforts.
+Overall, the community response was overwhelmingly positive, and led to some very fruitful discussions. We would like
+to thank all members of the community who participated. Some interesting points were brought to our attention, particularly
+when it came to the range of licenses compatible with [LGPL v3+][LGPL v3]. This raised a lot of questions about the
+suitability of [LGPL v3+][LGPL v3], and the current status of the MDAnalysis [GPL v2+][GPL v2] package license, so we
+decided to pause the relicensing process whilst we worked with our legal counsel to find an appropriate path forward.
+
+After careful consideration we have decided to aim for a license change to the [LGPL v2.1+][LGPL v2.1] via interim
+[package license](#package-license) changes to [GPL v3+][GPL v3] and [LGPL v3+][LGPL v3].
+
+In this post we will provide our rationale for this change, alongside key information for how we will be
+enacting these changes.
+
+**License changes will affect how users and developers will interact with the MDAnalysis code base going
+forward. We urge all members of the community to read this post and [get in touch](#package-license) with
+the MDAnalysis core developer team should you have any questions or concerns**.
+
+## Rationale for our change in target license
+
+*Table 1: MDAnalysis’ view of the dynamic linking license compatibility matrix. License pairs
+detailed with a compatibility of “No” have claimed incompatibilities between their license terms.
+License pairs detailed as “Yes ([copyleft](#copyleft])” are compatible but may incur copyleft through the
+interacting GNU GPL license. License pairs detailed with the compatibility “Yes” are, to our knowledge,
+fully compatible without incurring any [copyleft](#copyleft) penalty.*
```

Comment:
I wouldn’t call it “penalty” — just state the fact without adding an opinion — others might not see it as a penalty